add logo blur option

bob8677 opened the pull request:
I removed the commented out grayscale part - did not seem to be used

maybe change the iconf name - I'm not sure and just chose something as a example

bob8677 committed:
add logo blur option

paperbenni commented:
If we're going to implement choosing different effects then why not keep the option of making it black and white and give it an iconf value?
The iconf name is fine btw :+1:

bob8677 committed:
make logo effects more scalable + add more options

bob8677 commented:
I rewrote it to allow for more options and added a bunch. Some might not be wanted, not sure.

Force pushing was to remove commits fixing codacy issues.

bob8677 committed:
set effect to invert if not set
